primes.rb

```diff
@@ -1,6 +1,16 @@
+require 'prime'
+
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
+  def self.sum_to(num)
+    Prime.each(num).inject(&:+)
```

Comment:
Right logic, but we'd be able to teach you some tricks that make this less memory intensive. Plus some stying tips to make the code more Ruby like. Good effort though!